### ISSUE TITLE
ios onscreen keyboard and address bar issue

### ISSUE THREAD

**Issue by redeye86** [samclarke/SCEditor#600]
What happens:
On ios Safari and Google Chrome the onscreen keyboard cannot be closed and nothing can be typed into address bar.

How to reproduce:
1) Open https://www.sceditor.com/ using an ios device
2) Enter some text into the editor field
3) a) Try to hide the keyboard using the "Done" button (iPhone) / Keyboard down (iPad). Does not work or at least reappears immediately.
b) Click the address bar. On safari the recently-visited-overlay appears but text is still entered in the editor below that overlay and user is unable to leave the website

It seems that the editor gets focus immediately after blur. If you click on another HTML element outside the editor first, you can proceed. But especially on mobile websites you often do not have much free space around the editor and it is confusing.

Version affected v2.1.0

**Comment by redeye86** [comment]
I found a workaround. It seems that the editor is focussed immediately after blur when in wysiwyg mode; source mode is fine as it is not using an iframe. I fixed it by setting the read-only-mode for a short moment. It is disabled again by a setTimeout-function that is executed afterwards. So it is added to the execution queue and therefore runs after the focus event.

I added this fragment to handleEvent function:

```
[..]
var name = (e.target === sourceEditor ? 'scesrc' : 'scewys') + e.type;

>if(name=="scewysblur"){
>	base.readOnly(true);
>	window.setTimeout(function(){
>		base.readOnly(false);
>	});
>}

if (eventHandlers[name]) {
[..]
```

I am not sure if this should be used as a fix for that issue as it only relieves the symptoms.
